### DISCUSSION TITLE
SSL cert rotation

### DISCUSSION THREAD

**Discussion by dbason** [seaweedfs/seaweedfs#5816] [Q&A] [answered]
We are running seaweedfs on kubernetes.  In the filer pods we have a secret mounted with ssl certificates that are generated by cert-manager.  If the content of the ssl cert gets updated (e.g the certs get renewed before they expire) will filer automatically reload the new certs?

**Comment by kmlebedev** [comment, marked as answer]
Now certificates are automatically updated for https and grpc of all services

https://github.com/seaweedfs/seaweedfs/pull/4506
https://github.com/seaweedfs/seaweedfs/pull/5764/files#diff-36a7628e140641b032679eafe1503bd097ae9e2f76dd4d9ca0ba23d8e9670bb9R229
